pkg/distributor/otel.go

```diff
@@ -323,21 +323,22 @@ func newOTLPParser(
 		convOpts := conversionOptions{
 			addSuffixes:                       translationStrategy.ShouldAddSuffixes(),
 			enableCTZeroIngestion:             enableCTZeroIngestion,
-			enableStartTimeQuietZero:          enableStartTimeQuietZero,
 			keepIdentifyingResourceAttributes: keepIdentifyingResourceAttributes,
 			convertHistogramsToNHCB:           convertHistogramsToNHCB,
 			promoteScopeMetadata:              promoteScopeMetadata,
 			promoteResourceAttributes:         promoteResourceAttributes,
 			allowDeltaTemporality:             allowDeltaTemporality,
 			allowUTF8:                         !translationStrategy.ShouldEscape(),
 		}
-		metrics, metricsDropped, err := otelMetricsToTimeseries(
+
+		metrics, metadata, metricsDropped, err := otelMetricsToSeriesAndMetadata(
```

Comment:
[Nit] Unnecessary change
```suggestion
		metrics, metadata, metricsDropped, err := otelMetricsToSeriesAndMetadata(
```